code/modules/projectiles/guns/energy/pulse.dm

```diff
@@ -24,11 +24,11 @@
 /obj/item/weapon/gun/energy/pulse_rifle/destroyer
 	name = "pulse destroyer"
 	desc = "A heavy-duty, pulse-based energy weapon. Because of its complexity and cost, it is rarely seen in use except by specialists."
-	cell_type = /obj/item/weapon/cell/super
-	fire_delay = 25
+//	cell_type = /obj/item/weapon/cell/super
+	fire_delay = null		//Aspawn only gun doesn't really need to be balanced against the station/antags

```

Comment:
Not sure why you're doing this. Just because it's not obtainable in the game right now doesn't mean it should be made spammable - especially not for potential downgrade servers, a scenario where an admin is spawning it for the players, or when this gun will indeed be made constructable by a future commit.